File: src/main/java/me/rsman/BetterMinecraftCore/Managers/TasksManager.java
package me.rsman.BetterMinecraftCore.Managers;

import me.rsman.BetterMinecraftCore.BetterMinecraftCore;
import me.rsman.BetterMinecraftCore.Tasks.PlayerTasks;
import org.bukkit.plugin.Plugin;
import org.bukkit.scheduler.BukkitScheduler;

public final class TasksManager {
    public static void registerAllTasks(){
        BukkitScheduler scheduler = BetterMinecraftCore.getInstance().getServer().getScheduler();
        Plugin plugin = BetterMinecraftCore.getInstance();

        scheduler.runTaskTimerAsynchronously(plugin, PlayerTasks::updatePlayers, 0, 10);
    }
}

File: src/main/kotlin/me/rsman/BetterMinecraftCore/Managers/TasksManager.kt
package me.rsman.BetterMinecraftCore.Managers

import me.rsman.BetterMinecraftCore.BetterMinecraftCore
import java.lang.Runnable
import me.rsman.BetterMinecraftCore.Tasks.PlayerTasks
import org.bukkit.plugin.Plugin

object TasksManager {
    fun registerAllTasks() {
        val scheduler = BetterMinecraftCore.instance.server.scheduler
        val plugin: Plugin = BetterMinecraftCore.instance
        scheduler.runTaskTimerAsynchronously(plugin, Runnable { PlayerTasks.updatePlayers() }, 0, 10)
    }
}